scripts/process_notebooks.py

```diff
@@ -139,71 +139,49 @@ def should_process(path):
         # Clean the original notebook and save it to disk
         print(f"Writing complete notebook to {nb_path}")
         with open(nb_path, "w") as f:
-            nb_clean = clean_notebook(nb)
+            if nb_path.startswith("tutorials"):
+               flag = True
```

Comment:
Use a more informative variable name here so it's clear what the code is doing (e.g. `clear_outputs`).

Reply:
`clear_outputs` is a very informative choice. Thank you